### ISSUE TITLE
Separate project, but you guys could help

### ISSUE THREAD

**Issue by tzusman** [quangogster/todd-app#4]
On a different project, I'm using [this package](
https://github.com/ericgio/react-bootstrap-typeahead).  I would like to fork and add the following:

*This is all relevant to when multiple selections are enabled*
Go [here](http://ericgio.github.io/react-bootstrap-typeahead/) and click on `Allow multiple selections (tokenizer)`

- [ ] Add a prop that would be the placeholder of the input after one or more are already selected.  As you can see, after you select one, the input goes blank and it's not 100% clear that you can click on that area to select more.
- [ ] If you filter down so that there is only one item remaining in the dropdown, you should be able to hit enter and have it selected.  Currently you need to press down arrow then enter to select.

Cost to do these two?

**Comment by danielr18** [comment]
About the first one, do you mean something like this?

![image](https://cloud.githubusercontent.com/assets/9046616/22839825/68d155b0-efa1-11e6-804d-c787e6fd40ae.png)

**Comment by tzusman** [comment]
Yep exactly, it would just be a different text from the original placeholder.  Right now it says `Choose a state` then the other placeholder would be like `Choose additional states`

**Comment by danielr18** [comment]
We can do both for $75.

**Comment by tzusman** [comment]
k, just submitted on Upwork

**Comment by danielr18** [comment]
@tzusman features added, run npm install and then npm run example. You can open the index.html there and test them in the basic example.

**Comment by tzusman** [comment]
Sweet! Quick turn around. I'm out for the next couple hours. I'll take a look.

**Comment by tzusman** [comment]
Wow. fast and perfect.  Just released the funds on upwork.  Thanks!

**Comment by tzusman** [comment]
PR'ing it back to the repo in case they want it!

https://github.com/ericgio/react-bootstrap-typeahead/pull/136
